api-gateway-config/scripts/lua/routing.lua

```diff
@@ -59,10 +59,15 @@ function _M.processCall()
       end
       -- Parse backend url
       local u = url.parse(opFields.backendUrl)
+      -- add http:// if no protocol is specified
+      if u.host == nil then
```

Comment:
@codymwalker ah good catch - it's been fixed. Also fixed a little bug in the management interface for updating APIs with `/` basePath.